productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
```

Comment:
check the latest config files on `family-ess` repo to see if the schema has been updated.

Reply:
No updated version availale